index.html

```diff
@@ -161,7 +164,8 @@ <h3>Skills</h3>
         </section>
         <footer class="footer">
             <div class="contact-me-headline">
-            <h3 class="generic section-headline">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+            <div class="contact-form-headline-form">
+                <h3 class="generic section-headline contact-form-headline">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
             <form action="https://formspree.io/f/xlezlydw" method="post">
                 <ul class="contact-form-control">
```

Comment:
- [ ] In **Desktop** version the `Full name` section is divided into two parts , `First name` & `Last name`. Kindly add this in the contact section.
-
![Screenshot from 2022-03-25 00-25-36](https://user-images.githubusercontent.com/26767383/159991903-7c23725a-29d3-4537-89c9-a15c6647903a.png)

Reply:
@Deepakdanger Thank very much for your kind feedback and response, but in this project requirements says the next:

![image](https://user-images.githubusercontent.com/18579425/159993839-bf454440-bfa6-4f38-92f2-bb04e608b943.png)

Reply:
@jrosario19, let's make it optional :+1: